src/ur3e_hande_robot_description/config/initial_positions.yaml

```diff
@@ -0,0 +1,7 @@
+shoulder_pan_joint: 0.0
+shoulder_lift_joint: -1.57
+elbow_joint: 0.0
+wrist_1_joint: -1.57
+wrist_2_joint: 0.0
+wrist_3_joint: 0.0
+joint_finger: 0.01
```

Comment:
Can you explain why this joint finger is defaulted the way it is. I'm sure it's right, but I'm just surprised.

Reply:
joint_finger is a joint that connects the left finger to the base of the gripper. Initializing this joint is not very meaningful.